### PR TITLE
Add rule to prevent direct usage of $_GET, $_POST, $_REQUEST, $_FILES and $_COOKIE

### PR DESCRIPTION
**Pull request by arkener**
Reference: https://www.drupal.org/project/coder/issues/2958175

### CODE REVIEW

**Comment by arkener** [comment]
Apparently Travis CI failed to retrieve PHP7.4. Re-triggering to see if it's a one time thing.

File: coder_sniffer/DrupalPractice/Sniffs/Variables/GetRequestDataSniff.php [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+namespace DrupalPractice\Sniffs\Variables;
+
+use DrupalPractice\Project;
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+
+/**
+ * Ensures that Symfony request object is used to access super globals.
+ *
+ * Checks the usage of @author tags.
```
Comment (klausi) [inline coder_sniffer/DrupalPractice/Sniffs/Variables/GetRequestDataSniff.php:19]: wrong comment?

And we lost the useful comment from @alexpott in https://www.drupal.org/files/issues/2018-04-03/2958175-4.patch where this sniff came from.

**Review by klausi — COMMENTED** [review]
Looks good, we just lost a useful comment. Will fix this manually and then merge this.

**Comment by klausi** [comment]
And merged, thanks!

**Pull request closed without merging**